config/routes.rb

```diff
@@ -30,6 +30,7 @@
   get '/login', to: "sessions#new"
   post '/login', to: "sessions#create"
   delete '/logout', to: "sessions#destroy"
+  get '/bookmark_notice', to: 'sessions#bookmark_notice'
```

Comment:
Should we just create a new controller for bookmarks?

Reply:
or add logic to user_videos controller create action?

Reply:
it's in user_videos now